test/vaadin-list-mixin.html

```diff
@@ -281,6 +281,47 @@
 
         beforeEach(() => list._focus(0));
 
+        describe('RTL mode', () => {
+
+          beforeEach(() => list.setAttribute('dir', 'rtl'));
+
+          it('should move focus to next element on "arrow-down" keydown', () => {
+            arrowDown(list);
+            expect(list.items[1].focused).to.be.true;
+          });
+
+          it('should move focus to prev element on "arrow-up" keydown', () => {
+            arrowDown(list);
+            arrowUp(list);
+            expect(list.items[0].focused).to.be.true;
+          });
+
+          it('should move focus to prev element on "arrow-left" keydown', () => {
+            list.orientation = 'horizontal';
+            arrowLeft(list);
+            expect(list.items[1].focused).to.be.true;
+          });
+
+          it('should move focus to next element on "arrow-right" keydown', () => {
```

Comment:
Please change the test suite description, should be `prev element`.

Reply:
`next` element was here in the meaning of the `visually previous` element (not the DOM order it is visual tests).
But if it easier this way, updated.